en/06_Project_Governance/05_Major_release_policy.md

```diff
@@ -0,0 +1,135 @@
+
+Only one major release line is in "active development" at once.
+
+### Bug and security fixes
+
+A Silverstripe CMS major release line enters the "bug and security fixes" phase once the follow major release is tagged stable.
+
+Release lines in the "bug and security fixes" phase will receive:
+- patches for low impact bugs
+- patches for low low severity security issues. 
+
+Release lines in the "bug and security fixes" phase will not receive:
+- new feature or new APIs
+- new minor releases.
+
+A major release line stays in the "bug and security fixes" phase for 1 year.
+### Security fixes only
+
+At the end of "Bug and security fixes" phase, a Silverstripe CMS major release line transition to the "security fixes only" phase.
+
+In the "security fixes only" phase, a Silverstripe CMS major release line will only receive patches for high impact security issues defined has having a CVSS of 7.0 or above.
+
+### End-of-life
+
+A Silverstripe CMS major release line will be end-of-life once its follow up major release line exit active development.
+
+End-of-life major release do not receive any security fixes or bug fixes. 
+
+## PHP Support commitments
+
+Silverstripe CMS major releases tracks PHP releases. 
+
+The Silverstripe CMS release cycle is built around the assumptions that:
+- New PHP releases are shipped at the end of November on an annual basis.
+- PHP releases are in full support for two years with 1 year of limited support. 
+
+At launch, a Silverstripe CMS major release support all PHP version in full support. PHP versions in limited support are not supported at launch by new Silverstripe CMS majors.
+
+Following the initial release of a Silverstripe CMS major, we will aim to add forward compatibility for the next PHP release. e.g.: Silverstripe CMS 5 at launch will support PHP 8.1 and PHP 8.2. Will aim to add support for an eventual PHP 8.3 release to CMS 5.
+
+We try our best not to drop support for older PHP releases within Silverstripe CMS major release line even if those PHP release reach end-of-life.
+
+## Dependency management
+
+At launch new Silverstripe CMS majors aim to have all PHP and JS dependencies on their latest stable version. For dependencies offering a *Long Term Support* version, the latest LTS of that dependency will be preferred.
+
+The Silverstripe CMS development team will define a list of "fixed dependencies" for each Silverstripe CMS major release line. *Fixed dependencies* will not changed for the lifetime of a Silverstripe CMS major release line.
+
+Support for newer major releases of fixed dependencies will not be retroactively added to Silverstripe CMS major release line, even if the currently used version is end-of-life. The only exception is if the upgrade is necessary to fix a high impact bug or security vulnerability. We’ll only support 1 major release line of each fixed dependency.
```

Comment:
We'll be running one year behind Symfony. Presuming, people are not in a desperate hurry to jump on the latest major Silverstripe CMS, that should give other vendor a reasonable chance to add support for newer symfony releases.

Symfony is a bit special and there could be a case for treating it a bit like we treat PHP. e.g.: Support for all fully supported majors or add support for new majors as they are release.

Maybe this is something we can revisit once Symfony 7 is released. I would rather start with a stricter support model and loosen it later, rather than start with a loose approach and discover its a major pain in the backside.

Reply:
Fair enough, I'm okay revisiting this when we have some real world usage of this policy to reflect on.